config.go

```diff
@@ -233,6 +233,8 @@ type Config struct {
 
 	AcceptKeySend bool `long:"accept-keysend" description:"If true, spontaneous payments through keysend will be accepted. [experimental]"`
 
+	KeySendHoldTime time.Duration `long:"keysend-hold-time" description:"If non-zero, keysend payments are accepted but not immediately settled. If the payment isn't settled manually after the specified time, it is canceled automatically. [experimental]"`
```

Comment:
```suggestion
	KeySendHoldTime time.Duration `long:"keysend-hold-time" description:"If non-zero, keysend payments are accepted but not settled immediately settled. If the payment isn't settled manually after the specified time, it is canceled automatically. [experimental]"`
```

Reply:
"not settled immediately settled" -> how is this an improvement?

Reply:
oops somehow i missed the existing _settled_ haha. i read "keysend payments are accepted but not immediately", ignore